library/src/lib/table/table.module.ts

```diff
@@ -1,12 +1,13 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 
-import { TableComponent } from './table.component';
+import { TableDirective } from './table.component';
 import { ColumnSortableDirective } from './column-sortable.directive';
+import { CdkTableModule } from '@angular/cdk/table';
 
 @NgModule({
-    imports: [CommonModule],
-    declarations: [TableComponent, ColumnSortableDirective],
-    exports: [TableComponent, ColumnSortableDirective]
+    imports: [CommonModule, CdkTableModule],
```

Comment:
should not import CdkTable module in the lib itself, it should only be for the docs